clusterloader2/pkg/chaos/monkey.go

```diff
@@ -35,15 +39,27 @@ func NewMonkey(client clientset.Interface, provider string) *Monkey {
 
 // Init initializes Monkey with given config.
 // When stopCh is closed, the Monkey will stop simulating failures.
-func (m *Monkey) Init(config api.ChaosMonkeyConfig, stopCh <-chan struct{}) error {
+func (m *Monkey) Init(config api.ChaosMonkeyConfig, stopCh <-chan struct{}) (*sync.WaitGroup, error) {
```

Comment:
please rebase - these changes are already at head

Reply:
sorry i had rebased but probably against the wrong branch. this has been removed.